relayer/chains/icon/event_parse.go

```diff
@@ -85,10 +85,12 @@ func (p *Provider) parseCallMessage(e *types.EventLog, eventType string, height
 
 // Parse Event
 func (p *Provider) parseMessageEvent(notifications *types.EventNotification) ([]*providerTypes.Message, error) {
-	height, err := notifications.Height.BigInt()
+	aboveHeight, err := notifications.Height.BigInt()
 	if err != nil {
 		return nil, err
 	}
+	height := aboveHeight.Sub(aboveHeight, big.NewInt(1))
```

Comment:
It is unnecessary to subtract the progressed height as it has been already progressed.

Reply:
The event is published one height above the height on which the transaction is included. So, to get the desired height we have to subtract the height of event notification by one.